File: twiml/voice/connect/conversationrelay/conversationrelay-1/conversationrelay-1.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
response.connect do |connect|
    connect.conversation_relay(url: 'wss://mywebsocketserver.com/websocket', language: 'sv-SE') do |conversationrelay|
        conversationrelay.language(code: 'sv-SE', tts_provider: 'amazon', voice: 'Elin-Neural', transcription_provider: 'google', speech_model: 'long')
        conversationrelay.language(code: 'en-US', tts_provider: 'google', voice: 'en-US-Journey-O')
end
end

puts response

File: twiml/voice/connect/conversationrelay/conversationrelay-1/conversationrelay-1.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var connect = new Connect();
        var conversationrelay = new ConversationRelay(url: "wss://mywebsocketserver.com/websocket", language: "sv-SE");
        conversationrelay.Language(code: "sv-SE", ttsProvider: "amazon",
            voice: "Elin-Neural", transcriptionProvider: "google",
            speechModel: "long");
        conversationrelay.Language(code: "en-US", ttsProvider: "google",
            voice: "en-US-Journey-O");
        connect.Append(conversationrelay);
        response.Append(connect);

        Console.WriteLine(response.ToString());
    }
}
